.golangci.yml

```diff
@@ -108,7 +108,7 @@ linters-settings:
         arguments:
           - maxLitCount: "3"
             allowStrs: '""'
-            allowInts: "0,1,2,3,4"
+            allowInts: "0,1,2,3,4,5,6,7"
```

Comment:
I would prefer we don't do this and instead we do like we did here:
https://github.com/cloudposse/atmos/pull/1061/files#diff-4150a187275800fb2386aef6bc6821fc38b10327e5a0def217338f97bc53d7e5R126-R132

For example, had we added the lint rule change proposed in this PR, then this other PR would also not have likely added constants - but we absolute should.

Reply:
Or we dismiss the lint alert as won't fix.